src/components/NavigationTabs/index.tsx

```diff
@@ -80,7 +79,9 @@ export function FindPoolTabs({ origin }: { origin: string }) {
         <HistoryLink to={origin}>
           <StyledArrowLeft />
         </HistoryLink>
-        <ActiveText>Import Pool</ActiveText>
+        <ActiveText>
+          <Trans id="nav.importPools">Import Pools</Trans>
```

Comment:
Might have added an extra 's' here, or perhaps the original text misses the 's'?

Reply:
nice catch!